build.sbt

```diff
@@ -271,12 +260,13 @@ lazy val zincCompileCore = (project in internalPath / "zinc-compile-core")
 //   format from which Java sources are generated by the sbt-contraband plugin.
 lazy val compilerInterface = (project in internalPath / "compiler-interface")
   .enablePlugins(ContrabandPlugin)
-  .disablePlugins(com.typesafe.sbt.SbtScalariform)
   .settings(
     minimalSettings,
     // javaOnlySettings,
     name := "Compiler Interface",
-    crossScalaVersions := Seq(scala212),
+    // Use the smallest Scala version in the compilerBridgeScalaVersions
```

Comment:
I thought you did this so that the regression doesn't happen again. Forcing the latest Java version will always guarantee that our bridge compiles for all the Java versions supported, I presumed that was what you had in mind.